src/tests/integration/test-data/src/Unit.t.sol

```diff
@@ -4,44 +4,173 @@ pragma solidity ^0.8.13;
 import {Test, console} from "forge-std/Test.sol";
 
 contract UnitTest is Test {
-    function test_assertEq_0() public pure {
+    function test_assertEq() public pure {
         assertEq(uint256(11), 11);
     }
 
-    function test_assertEq_1(address a, address b) public pure {
+    function test_assertEq(address a, address b) public pure {
         vm.assume(a == b);
         assertEq(a, b);
     }
 
-    function test_assertEq_2(bool a, bool b) public pure {
+    function test_assertEq(bool a, bool b) public pure {
         vm.assume(a == b);
         assertEq(a, b);
     }
 
-    function test_assertEq_3(bytes32 a, bytes32 b) public pure {
+    function test_assertEq(bytes32 a, bytes32 b) public pure {
         vm.assume(a == b);
         assertEq(a, b);
     }
 
-    function test_assertTrue(bool value) public pure {
-        vm.assume(value == true);
-        assertTrue(value);
+    function test_assertEq(int256 a, int256 b) public pure {
+        vm.assume(a == b);
+        assertEq(a, b);
+    }
+
+    function test_assertEq_err() public {
+        string memory expected = "throw test: 11 != 121";
+        string memory err = "throw test";
+        vm.expectRevert(bytes(expected));
+        assertEq(uint256(11), 121, err);
```

Comment:
Is it worth adding a test that checks that a failing `assertEq` not taking an `err` argument reverts and returns `"assertion failed"`?

Reply:
Good idea! I'll add the test now.

Reply:
Added the test in commit https://github.com/runtimeverification/kontrol/pull/558/commits/2d15e71fad09a071a32a0833e92cef53a9903862. I also added it to the skipped list because it is blocked by https://github.com/runtimeverification/kontrol/issues/556.